### DISCUSSION TITLE
接收群发的图片

### DISCUSSION THREAD

**Discussion by 347180440** [DreamVoid/MiraiMC#460] [问答 / Q&A]
将图片返回 或者得到图片的分辨率 或保存图片 的方法

**Comment by DreamVoid** [comment]
mirai 本身不支持获取图片文件，建议寻求 mirai 的帮助：https://github.com/mamoe/mirai

**Comment by 347180440** [comment]
https://github.com/mamoe/mirai/discussions/2816#discussioncomment-7443349

**Reply by 347180440** [reply]
可以更新这个方法吗

**Reply by DreamVoid** [reply]
现实事务繁忙，如果你有时间欢迎帮我写一个封装，或者自行调用mirai的方法，或者等。

**Reply by 347180440** [reply]
我并不会 我也只是个小白 我还是等您有空在封装吧
